src/qml/FeatureForm.qml

```diff
@@ -242,8 +242,13 @@ Page {
           property var widget: EditorWidget
           property var field: Field
           property var fieldType: FieldType
+          property var relationId: RelationId
           property var constraintValid: ConstraintValid
-
+          property var relationEditorModel:  ReferencedFeatureListModel {
```

Comment:
I would probably put this part into the widget (and just pass relation id, maybe even as part of the configuration?)